src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/table.scss

```diff
@@ -0,0 +1,113 @@
+@import '../../containers/Application/colors.scss';
+
+$headerTextColor: $white;
+$headerCellBackgroundColor: $blueZodiac;
+$headerControlCellTextColor: rgba($silverChalice, .25);
+$headerControlCellBorderColor: rgba($silverChalice, .25);
+$cellTextColor: $blueZodiac;
+$cellTextColorActive: $white;
+$cellBackgroundColor: $white;
+$cellBorderColor: $cellBackgroundColor;
+$cellBorderColorActive: $shakespeare;
+$cellBackgroundColorActive: $shakespeare;
+$controlCellBorderColor: $white;
+
+.tableContainer {
+    overflow: scroll;
+}
+
+.table {
+    border-collapse: separate;
+    border-spacing: 0 5px;
+    margin: -5px 0;
+    overflow: hidden;
+}
+
+.header {
+    color: $headerTextColor;
+
+    .row {
+        height: 60px;
+    }
+}
+
+.header-cell {
+    padding: 20px;
+    text-align: center;
+    font-size: 12px;
+    font-weight: normal;
+    background-color: $headerCellBackgroundColor;
+
+    &:first-child {
+        border-radius: 3px 0 0 3px;
+    }
+
+    &:last-child {
+        border-radius: 0 3px 3px 0;
+    }
+}
+
+.header-control-cell {
```

Comment:
There are currently no `control-cell`s, but still a few lines of css are lying around (not only this one).